scripts/agent-trader/agent_trader.py

```
@@ -232,10 +282,10 @@ def inline(t):
             if in_ul:
                 out.append("</ul>"); in_ul = False
             out.append(f"<h3 style='margin:12px 0 4px'>{inline(l[3:])}</h3>")
-        elif re.match(r"^\s*(?:[-*]|\d+\.)\s+", l):
+        elif _LIST_ITEM.match(l):
             if not in_ul:
                 out.append("<ul style='margin:4px 0;padding-left:20px'>"); in_ul = True
```

Ruff (0.16.0)

[error] Multiple statements on one line (semicolon)

(E702)